### DISCUSSION TITLE
Flows: modify "step" data

### DISCUSSION THREAD

**Discussion by dkamali-mc** [directus/directus#14283] [Archive]
Hello everyone, 

In many ETL platforms, you have the ability to modify/transform data before proceeding to other steps, is it possible to use JS to modify the data, for example, increment values (calculate tax)? 

Thanks

**Comment by azrikahar** [comment]
Currently it is not possible to use JS to do increments/calculations (the [Transform Payload](https://docs.directus.io/configuration/flows/operations/#transform-payload) operation can only combine/pick/merge data for now), but this would be a great feature indeed and was brought up in #13944 #13889. Feel free to give #13944 an upvote 👍
